Dockerfile

```diff
@@ -54,6 +53,7 @@ RUN cd $GOPATH/src/github.com/flannel-io/cni-plugin && \
 WORKDIR $GOPATH/src/github.com/containernetworking/plugins
 RUN xx-verify --static bin/*
 RUN go-assert-static.sh bin/* && \
+    export ARCH=$(xx-info arch) && \
```

Comment:
In this other PR https://github.com/rancher/image-build-addon-resizer/pull/15/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557 we are changing ARCH with TARGETARCH in the Dockerfile, is it correct that we are not doing this here?

Reply:
This change is not directly related to the other changes in the pull request. 

While testing, I noticed that the case to run go-assert-boring was never reached, so I re-exported this variable to match its use earlier in the dockerfile.

Reply:
TARGETPLATFORM is still being used in the same way as it was with the docker/build-and-push action.